05-Mining-Protocol.md

```diff
@@ -1,106 +1,195 @@
-## 5.1 Channels
+# 5. Mining Protocol
 
-The protocol is designed such that downstream devices (or proxies) open communication channels with upstream stratum nodes within established connections.
-The upstream stratum endpoints could be actual mining servers or proxies that pass the messages further upstream.
-Each channel identifies a dedicated mining session associated with an authorized user.
-Upstream stratum nodes accept work submissions and specify a mining target on a per-channel basis.
+The Mining Protocol is used to distribute work to Mining Devices, and it's also how they submit their proof of work.
 
-There can theoretically be up to 2^32 open channels within one physical connection to an upstream stratum node.
-All channels are independent of each other, but share some messages broadcast from the server for higher efficiency (e.g. information about a new prevhash).
-Each channel is identified by its `channel_id` (`U32`), which is consistent throughout the whole life of the connection.
+It can be used without Job Declaration and Template Distribution Protocols, where a subset of the messages are used by a pool to unilaterally distribute work to miners.
 
-A proxy can either transparently allow its clients to open separate channels with the server (preferred behavior) or aggregate open connections from downstream devices into its own open channel with the server and translate the messages accordingly (present mainly for allowing v1 proxies).
-Both options have some practical use cases.
-In either case, proxies SHOULD aggregate clients' channels into a smaller number of TCP connections.
-This saves network traffic for broadcast messages sent by a server because fewer messages need to be sent in total, which leads to lower latencies as a result.
-And it further increases efficiency by allowing larger packets to be sent.
+Alternatively, it can be used in conjunction with Job Declaration and Template Distribution Protocols, where both pool and miners coordinate to decide what is valid work.
 
-The protocol defines three types of channels: **standard channels**, **extended channels** (mining sessions) and **group channels** (organizational), which are useful for different purposes.
+## 5.1 Jobs
```

Comment:
if you look at the current state of the Mining Protocol specs (before this PR), 

every other section name is plural, so this is just the original pattern:
- `## 5.1 Channels`
- `### 5.1.1 Standard Channels`
- `### 5.1.2 Extended Channels`
- `### 5.1.3 Group Channels`
- `### 5.1.4 Future Jobs`

I see your point about singular being more appropriate as a way to define some concept, but the proper way to fix this will be to change everything else into singular as well

Reply:
ah nevermind, I see you also made suggestions for the other titles as well